lib/src/emoji_picker.dart

```diff
@@ -269,6 +280,12 @@ class EmojiPickerState extends State<EmojiPicker> {
           );
       }
 
+      if (widget.scrollController != null) {
+        final autoScrollController = widget.scrollController!;
+        autoScrollController
+            .jumpTo(autoScrollController.position.maxScrollExtent);
+      }
+
```

Comment:
Would it make sense to put it after `widget.onEmojiSelected?.call(category, emoji);` because if someone uses `TextInputController` in `onEmojiSelected` with then it wouldn't scroll?

Reply:
I made a [commit](https://github.com/Fintasys/emoji_picker_flutter/pull/183/commits/4be65c96d36140d0602327596d2b1020688f3f26) to version 2.0 addressing the same issue based on your code. 
If you want to add the feature to v1.x of the emoji picker then it would be nice if you could check my commit and adjust your PR.

Reply:
> Would it make sense to put it after `widget.onEmojiSelected?.call(category, emoji);` because if someone uses `TextInputController` in `onEmojiSelected` with then it wouldn't scroll? Same question with ` widget.onBackspacePressed?.call();` above

Yes, Indeed you are right. I have update the PR based on your changes in version 2.0

Reply:
Thanks for quick response ! Sorry, but I had to added another fix to it, would be great if you could include them as well. https://github.com/Fintasys/emoji_picker_flutter/commit/8e4a2873d340a507400be0b433a0f7dea11307b1

Reply:
@Fintasys  I have updated the PR. Please review it. Thanks :)